src/common/AsyncOpTracker.cc

```diff
@@ -9,19 +9,19 @@ AsyncOpTracker::AsyncOpTracker()
 }
 
 AsyncOpTracker::~AsyncOpTracker() {
-  Mutex::Locker locker(m_lock);
+  std::lock_guard<Mutex> locker(m_lock);
```

Comment:
I don't have a horse in this race but my understanding is that `scoped_lock` should be favored over `lock_guard` in new code.

Also, I think that template argument deduction could be used so you don't need to explicitly write `<Mutex>`. Haven't tried though...